hardware/PhilipsHue/PhilipsHue.cpp

```diff
@@ -88,15 +103,16 @@ bool CPhilipsHue::StopHardware()
     return true;
 }
 
-#define HUE_POLL_INTERVAL 10
 
 void CPhilipsHue::Do_Work()
 {
 	int msec_counter = 0;
-	int sec_counter = HUE_POLL_INTERVAL-2;
+	int sec_counter = 0;
```

Comment:
i would make this

int sec_counter = m_poll_interval - 1;

and remove the below line after the log message